ironfish-rust-nodejs/src/structs/transaction.rs

```diff
@@ -521,8 +521,8 @@ pub fn aggregate_signature_shares(
     signing_package_str: String,
     signature_shares_arr: Vec<String>,
 ) -> Result<Buffer> {
-    let public_key_package = PublicKeyPackage::deserialize(
-        &hex_to_vec_bytes(&public_key_package_str).map_err(to_napi_err)?,
+    let public_key_package = PublicKeyPackage::read(
+        &hex_to_vec_bytes(&public_key_package_str).map_err(to_napi_err)?[..],
```

Comment:
I think this implies that each account will store the public key package as a full ironfish-frost `PublicKeyPackage` rather than a frost `PublicKeyPackage`

That seems fine to me, but wanted to note it